### PR TITLE
fixed #451

### LINKED ISSUE #451: [BUG] When hovered on footer links the cursor shape is different on some links
### Describe the bug

In the footer when hovered over 
About
Services
Community
Contribute
the cursor is set to auto So 
image
appears while in other links the cursor is set to Pointer 

### To Reproduce

It is an easy fix

1. Go to 'TheCyberHUB\src\components\Homepage\Footer\FooterElements.jsx'
2. Scroll down to `const FooterScrollLink`
3. Add CSS line `cursor: pointer;`

The issue is fixed

### Expected Behavior

After applying the fix
Now when we hover on 
About
Services
Community
Contribute
The pointer cursor will be showed
image
Now it will be uniform

### Screenshot/ Video

https://github.com/thecyberworld/TheCyberHUB/assets/75574622/e0452f0e-f979-46ce-815a-a1e7aa3477cf

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by abhayst007**
## Fixes Issue

Fixed issue #451 

## Changes proposed

Added CSS `line cursor: pointer; `to  `const FooterScrollLink `in  _'TheCyberHUB\src\components\Homepage\Footer\FooterElements.jsx'_

## Screenshots

image

## Code of Conduct

- [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/thecyberworld/Support/blob/main/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

---

You can also join our [Discord](https://discord.gg/QHBPq6xP5p) community.
Feel free to check out other cool repositories of the [Thecyberworld](https://github.com/thecyberworld).
Join the Thecyberworld GitHub Organisation by raising an [issue](https://github.com/thecyberworld/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will be sent an invitation).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "thecyberhub.org",
-      "version": "0.53.12",
+      "version": "0.53.14",
```
Comment (kabir0x23) [inline package-lock.json:9]:
```suggestion
      "version": "0.53.12",
```

we actually never push other files other than we have installed anything new in the packages 
can u fix this is package-lock.json

Comment (abhayst007) [reply package-lock.json:9]: Yes for sure

Comment (kabir0x23) [reply package-lock.json:9]: > Yes for sure

if u need any help or having any doubts, let us know

**Comment by abhayst007** [comment]
Updated package-locl.json

**Comment by abhayst007** [comment]
Hey @kabir0x23 why is the status blocked tag in this issue

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "homepage": "https://thecyberhub.org/",
   "name": "thecyberhub.org",
   "private": true,
-  "version": "0.53.14",
+  "version": "0.53.12",
```
Comment (kabir0x23) [inline package.json:5]:
```suggestion
  "version": "0.53.14",
```

**Pull request merged**